packages/clerk-js/src/ui/contexts/components/SignIn.ts

```
@@ -121,16 +120,31 @@ export const useSignInContext = (): SignInContextType => {
-    routing: ctx.routing,
-    baseUrl: signInUrl,
-    taskUrls: clerk.__internal_getOption('taskUrls'),
-  });
+  const navigateOnSetActive = async ({ session, redirectUrl }: { session: SessionResource; redirectUrl: string }) => {
+    const currentTask = session.currentTask;
+    if (!currentTask) {
+      return navigate(redirectUrl);
+    }
+
+    const taskEndpoint = getTaskEndpoint(currentTask);
+    const taskNavigationPath = isCombinedFlow ? '/create' + taskEndpoint : taskEndpoint;
```

ESLint

[error] 'taskNavigationPath' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)